### PR TITLE
fix: Publish zToken contract as an npm package and import into zNS

### PR DESCRIPTION
**Pull request by JamesEarle**

### CODE REVIEW

File: test/helpers/deployZNS.ts [hunk 1]
```diff
@@ -207,7 +243,7 @@ export const deployZNS = async ({
     deployer,
     accessController.address,
     priceOracle.address,
-    zeroTokenMock.address,
```
Comment (Whytecrowe) [inline test/helpers/deployZNS.ts]: it's still a mock. not actual zero token. let's specify this, so nobody deploys it by mistake to a network

File: package.json [hunk 2]
```diff
@@ -43,9 +43,10 @@
     "@types/mocha": "9.1.0",
     "@types/node": "^18.15.11",
     "@zero-tech/eslint-config-cpt": "0.2.7",
+    "@zero-tech/ztoken": "^1.0.1",
```
Comment (Whytecrowe) [inline package.json]: let's make a specific version here rather than caret

**Review by Whytecrowe — CHANGES_REQUESTED** [review]
couple of minor things

File: test/helpers/deployZNS.ts (hunk 1 above)
Comment (JamesEarle) [reply test/helpers/deployZNS.ts]: Makes sense, I'll add this as well as a comment specifying that this script is used for testing only

**Review by Whytecrowe — APPROVED** [review]

**Pull request merged**